server/src/project/project.service.ts

```diff
@@ -382,77 +387,85 @@ export class ProjectService {
       }, 1);
     const [firstProject] = projects;
 
-    const transformedProject = await transformProjectAttributes(firstProject);
-
-    // get geoms from carto that match array of bbls
-    const bblsFeaturecollection = await this.geometryService.getBblsFeaturecollection(firstProject.bbls);
-
-    if (bblsFeaturecollection == null) {
-      console.log(`MapPLUTO does not contain matching BBLs for project ${firstProject.id}`);
+    if (projects.length < 1) {
```

Comment:
Could also just say 

`if (projects.length < 1) throw new HTTP....`


No need for else block